R/utils.R

```diff
@@ -21,3 +21,10 @@ generate_id <- function () {
   paste0(paste0(sample(c(LETTERS, letters, 0:9), 5, TRUE),
                 collapse = ""), round(as.numeric(Sys.time()) * 1000))
 }
+
+#' Return description with datetime prefix.
+#' @param description character
+#' @return character
+format_description <- function(description) {
```

Comment:
What are the reasons to introduce such function?

Reply:
During testing I realised that if you run your validation pipeline multiple times, you will just accumulate error / success counter and it will be unclear if e.g. 6 violations mean 6 problematic entries / rows, or your run your code 2 times and had 3 violations.

But I think it is a discussion for another pull request. I am removing it for now and I would like to get back to it after we agree on the data.validator new API.

Reply:
Let's address it later then.